tests/test_code_report.py

```diff
@@ -61,25 +76,32 @@ def test_without_code_report_command(runner_with_pylint):
     assert not pattern.findall(log)
 
 
-@pytest.mark.parametrize('args, expected_log', [
-    [["--python-version", python_version(), "--files", "source_file.py", "--result-file", "${CODE_REPORT_FILE}", '--rcfile'],
-     'rcfile: expected one argument'],
+@pytest.mark.parametrize('analyzer', ['pylint', 'mypy'])
```

Comment:
pylint and mypy share a lot of input parameters, so it makes sense to group them like this